# We always want the system libffi for the Ubuntu package.

--- a/setup.py
+++ b/setup.py
@@ -1741,6 +1741,10 @@
             ext.libraries.append(ffi_lib)
             self.use_system_libffi = True
 
+        if not self.use_system_libffi:
+            print("Error: not using system libffi", file=sys.stderr)
+            sys.exit(1)
+
 
 class PyBuildInstall(install):
     # Suppress the warning about installation into the lib_dynload
